src/index.js

```diff
@@ -18,4 +18,22 @@ function render(ui, {container = document.createElement('div')} = {}) {
   }
 }
 
-export {render, Simulate, wait}
+function renderIntoDocument(ui) {
+  return render(ui, {
+    container: document.body.appendChild(document.createElement('div')),
+  })
+}
+
+function clearDocument() {
+  document.body.innerHTML = ''
+}
+
+// fallback to synthetic events for DOM events that React doesn't handle
```

Comment:
This comment needs an update I think. Unless I'm misunderstanding. It should say:

```
// fallback to synthetic events for React events that the DOM doesn't support
```